uk.co.compucorp.civicrm.hrsampledata/tests/phpunit/CRM/HRSampleData/Cleaner/HRHoursLocationTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+require_once EXTENSION_ROOT_DIR . 'CRM/HRSampleData/Importer/HRHoursLocation.php';
```

Comment:
mmmm strange .. I couldn't find any clue  .. but apparently I can remove all require_once and all tests works fine ( which is really really weird ) .